src/components/_variables.scss

```diff
@@ -10,4 +10,5 @@ $grey: #ccc !default;
 $charcoal: #2f2833 !default;
 $purple: #9a3cbb !default;
 $pink: #ff1966 !default;
+$pink2: #fc2268 !default;
```

Comment:
![image](https://user-images.githubusercontent.com/8657986/68600298-2b878d80-049a-11ea-838d-61e8bce48ab3.png)

These colours are almost impossible to differentiate. I don't think we should be adding the new one